### ISSUE TITLE
Is the visualization result normal?

### ISSUE THREAD

**Issue by WangYZ1608** [facebookresearch/mae#174]
Hello,

thank you for the great work and the great repo. 

I reproduced the pre-training of MAE-ViT-Large, performed 800 epochs, and set `norm_pix_loss=True`. Below is the visualization result and tensorboard curve, is this normal?

the result similar to #156

![result](https://github.com/facebookresearch/mae/assets/100399172/5b15041a-e17a-4777-b3cd-9453685746ed)
![Snipaste_2023-07-31_16-48-29](https://github.com/facebookresearch/mae/assets/100399172/05cc8bbb-9855-413a-ba22-e3874770f74f)

**Comment by jfcarre** [comment]
I am having similar results, any update?

**Comment by xywen97** [comment]
Almost the same promblem.  When I try to finetune the MAE model with MSE loss on a new small dataset, I found the loss was decreasing but the visualization results were quite strange (be like what you show above). So weired.

**Comment by ashley-ferreira** [comment]
I am running into this issue as well. I noticed on a closed issue it was mentioned that this might be because the checkpoints are only storing the weight of the encoder and not the decoder: https://github.com/facebookresearch/mae/issues/156. I haven't quite figured this out yet though.

**Comment by Burton123456** [comment]
I am having similar results, any update?

**Comment by c7878783** [comment]
I have the same problem with WangYZ1608, but I finally found the problem is caused by incorrectly load the pretrained model.So your visulize results may come from a randomly intialized model, not your pretrained model.
My loss is finally 0.3~0.4, my visualize results is colorful patch, not the results like mosaic.

**Comment by ShakespeareXu** [comment]
same problem, need help please
